packaging/ubuntu-16.04/rules

```diff
@@ -90,6 +90,16 @@ override_dh_auto_build:
 	cd cmd && ( ./configure --prefix=/usr --libexecdir=/usr/lib/snapd $(VENDOR_ARGS))
 	$(MAKE) -C cmd all
 
+	# Generate the real systemd units out of the available templates
+	cat data/systemd/snapd.service.in | \
+		sed s:@libexecdir@:/usr/lib:g | \
+		sed s:@SNAPD_ENVIRONMENT_FILE@:/etc/environment:g > data/systemd/snapd.service
+	cat data/systemd/snapd.refresh.service.in | \
+		sed s:@bindir@:/usr/bin:g | \
+		sed s:@SNAP_MOUNTDIR@:/snap:g > data/systemd/snapd.refresh.service
+	cat data/systemd/snapd.autoimport.service.in | \
```

Comment:
We don't have to. The packaging/ubtunu-14.04 ships its own version of the systemd units and doesn't install those from data/systemd